devtools/flake.nix

```diff
@@ -3,7 +3,7 @@
 
   inputs = {
     # change this path to the path of your roc folder
-    roc.url = "path:/home/username/gitrepos/roc9/roc";
+    roc.url = "path:/home/anton/gitrepos/roc";
```

Comment:
Should we leave this as `username` so people know this isn't a copy/paste error? 😄

Reply:
Sure, done :)